MAUI/DateTimePicker/events.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: post
+title: Events in .NET MAUI Date Time Picker control | Syncfusion
+description: Learn about events in Syncfusion .NET MAUI Date Time Picker (SfDateTimePicker) control and its basic features.
+platform: maui
+control: SfDateTimePicker
+documentation: ug
+---
+
+# Events in .NET MAUI Date Time Picker (SfDateTimePicker)
+
+## Selection changed event
+
+The [SelectionChanged] event is used to notify when the date time selection is changed onto the view in the [SfDateTimePicker].
+
+* `Sender`: This contains the `SfDateTimePicker` object.
+
+* `EventArgs`: In SfDateTimePicker picker, [DateTimePickerSelectionChangedEventArgs] is used for this event which holds the data of NewValue and OldValue.
```

Comment:
* `EventArgs`: In the SfDateTimePicker picker, the [DateTimePickerSelectionChangedEventArgs] is used for this event, which holds the data of NewValue and OldValue.